keyboards/ut47/keymaps/default/keymap.c

```diff
@@ -0,0 +1,121 @@
+/* Copyright 2018 Carlos Filoteo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "ut47.h"
```

Comment:
Ideally, this should be `#include QMK_KEYBOARD_H`

Reply:
I can do this, but I guess just a heads up that this was autogenerated by the util/new_project.sh script. Maybe that could use changes too.

Reply:
Actually, we just did.  There is a commit 2 days ago to update this.